codeforpdx/PASS
# Update package-lock dependencies to fix high-security vulnerability

NPM is reporting that a high security vulnerability was found in vite. Let's do a dependency update to fix that.

**Testing**
1. Pull Branch
2. Run `npm ci`
3. Check there are no vulnerabilities reported by npm during install
4. Smoke test the App
